### PR TITLE
TW-1768: Improve UI for chat screen

### PR DESCRIPTION
**Pull request by nqhhdev**
### Ticket

- #1768 

### DoD

- [x] update chat name correctly 
- [x] update profile name in greeting message in draft chat

https://github.com/linagora/twake-on-matrix/assets/99852347/30a44d53-b9df-45c0-a964-06aed08dca0c

- [x] mark as read action in chat list

https://github.com/linagora/twake-on-matrix/assets/99852347/6211b2f3-63ab-4e1b-b9ee-2ca04bd9fdbc

- [x] define `saas` environment for mobile [hardcode]

Case 1: platform is not `saas`

https://github.com/linagora/twake-on-matrix/assets/99852347/23e82555-c1f4-42d0-ac6a-6788019d8ea5

case 2: platform is `saas`

https://github.com/linagora/twake-on-matrix/assets/99852347/be5eb123-1f7a-4ca1-827b-5fc9e5277c83

### CODE REVIEW

**Comment by hoangdat** [comment]
- [ ] changing greeting message: include tag in greeting message

**Comment by nqhhdev** [comment]
> * [ ]  changing greeting message: include tag in greeting message

We will improve it other ticket #1777

File: docs/configurations/config_build_mobile_app_for_public_platform.md [hunk 1]
```diff
@@ -2,21 +2,23 @@
 
 ### Context
 
-- Twake Chat app can use the values of environment declarations to change its functionality or behavior. 
-  Dart compilers can eliminate the code made unreachable due to 
+- Twake Chat app can use the values of environment declarations to change its functionality or
```
Comment (hoangdat) [inline docs/configurations/config_build_mobile_app_for_public_platform.md:5]: What happens if user don't want it?

File: docs/configurations/config_web_app_for_public_platform.md [hunk 2]
```diff
@@ -22,10 +22,10 @@ in [config.sample.json](https://github.com/linagora/twake-on-matrix/blob/main/co
   "hide_redacted_events": false,
   "hide_unknown_events": false,
   "issue_id": "",
-  "registration_url": "https://example.com/",
-  "twake_workplace_homeserver": "https://example.com/",
+  "registration_url": "sampleValue",
+  "twake_workplace_homeserver": "sampleValue",
   "app_grid_dashboard_available": true,
-  "homeserver": "https://example.com/",
+  "homeserver": "sampleValue",
```
Comment (hoangdat) [inline docs/configurations/config_web_app_for_public_platform.md:28]: no need this change

**Review by hoangdat — APPROVED** [review]

**Pull request merged**